<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="5" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-06-08T17:01:49 IST" name="All Test Suite" finished-at="2024-06-08T17:02:09 IST" duration-ms="19887">
    <groups>
    </groups>
    <test started-at="2024-06-08T17:01:49 IST" name="LoginPageTestcase" finished-at="2024-06-08T17:02:07 IST" duration-ms="18262">
      <class name="scripts.LoginPage_Testcase">
        <test-method is-config="true" signature="onStart()[pri:0, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:01:49 IST" name="onStart" finished-at="2024-06-08T17:01:54 IST" duration-ms="4794" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onStart -->
        <test-method signature="test1()[pri:1, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:01:54 IST" name="test1" description="checking login functionality with empty username and password" finished-at="2024-06-08T17:01:59 IST" duration-ms="5689" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[testcase failed to log in to the application without username and password]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: testcase failed to log in to the application without username and password
at org.testng.Assert.fail(Assert.java:111)
at scripts.LoginPage_Testcase.test1(LoginPage_Testcase.java:60)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method signature="test2()[pri:2, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:01:59 IST" name="test2" description="checking login functionality with valid username and password" finished-at="2024-06-08T17:02:00 IST" duration-ms="1016" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method signature="test3()[pri:3, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:02:00 IST" name="test3" description="checking login functionality with some random usernames and passwords contains alphanumeric data from external excel.xlsx" finished-at="2024-06-08T17:02:06 IST" duration-ms="5590" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method signature="test4()[pri:4, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:02:06 IST" name="test4" description="verifying login through social network button available in login webpage" finished-at="2024-06-08T17:02:06 IST" duration-ms="364" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test4 -->
        <test-method is-config="true" signature="onFinish()[pri:0, instance:scripts.LoginPage_Testcase@79b06cab]" started-at="2024-06-08T17:02:06 IST" name="onFinish" finished-at="2024-06-08T17:02:07 IST" duration-ms="748" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onFinish -->
      </class> <!-- scripts.LoginPage_Testcase -->
    </test> <!-- LoginPageTestcase -->
    <test started-at="2024-06-08T17:02:07 IST" name="HomePageTestcase" finished-at="2024-06-08T17:02:09 IST" duration-ms="1586">
      <class name="scripts.HomePage_Testcase">
        <test-method is-config="true" signature="onStart()[pri:0, instance:scripts.HomePage_Testcase@50a638b5]" started-at="2024-06-08T17:02:07 IST" name="onStart" finished-at="2024-06-08T17:02:07 IST" duration-ms="62" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onStart -->
        <test-method signature="applicationLogin()[pri:1, instance:scripts.HomePage_Testcase@50a638b5]" started-at="2024-06-08T17:02:07 IST" name="applicationLogin" description="checking login functionality with valid username and password" finished-at="2024-06-08T17:02:08 IST" duration-ms="616" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- applicationLogin -->
        <test-method signature="homepage()[pri:2, instance:scripts.HomePage_Testcase@50a638b5]" started-at="2024-06-08T17:02:08 IST" name="homepage" description="checking homepage amount transaction table and verifying the sorted amount values" finished-at="2024-06-08T17:02:08 IST" duration-ms="482" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- homepage -->
        <test-method is-config="true" signature="onFinish()[pri:0, instance:scripts.HomePage_Testcase@50a638b5]" started-at="2024-06-08T17:02:08 IST" name="onFinish" finished-at="2024-06-08T17:02:09 IST" duration-ms="417" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- onFinish -->
      </class> <!-- scripts.HomePage_Testcase -->
    </test> <!-- HomePageTestcase -->
  </suite> <!-- All Test Suite -->
</testng-results>
